saviSimulation/savi/commandStation/CommandStationCore.java

```diff
@@ -17,10 +23,12 @@
 	 * @param connector 	The connector to the rest of the simulation
 	 * @param ID			The agent ID for the command station (address for messages)
 	 */
-	public CommandStationCore(CommandStationConnector connector, String ID) {
+	public CommandStationCore(String ID)  {
 		// Set message ID and connecter parameters
 		this.id = ID;
-		this.connector = connector;
+		this.socketConnector =  new SocketConnector(this, commandStationPort, simPort);
```

Comment:
Should probably also have an address here. Can be localhost for now (which I am assuming it is doing behind the scenes).

Reply:
Addressed by #157.